packages/enzyme/src/ReactWrapper.js

```diff
@@ -85,8 +85,13 @@ class ReactWrapper {
     const options = makeOptions(passedOptions);
 
     if (!root) {
+      const adapter = getAdapter(options);
+      if (!adapter.isValidElement(nodes)) {
```

Comment:
hmm, `nodes` could be an array of valid elements; i'm not sure we can check it directly.

also, i think we probably want to check this on `root` too if that's provided?

Reply:
I think if we try to `mount` an array of elements as the root that should still throw.

`root` elements should always have already been through this path when they were created, ie `wrap`/`ReactWrapper` constructor only get called with `null` when mounting (or diving in the shallow wrapper) or `this[ROOT]` when traversing, which is only set here.